common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -146,27 +149,44 @@
             },
 
             // @TODO move into utils.coffee
+            getNameWidth: function(name) {
+                var test = $('<div>'),
```

Comment:
@bjacobel: Below are the violations, and I don't see any of them mentioning `test`.  Most of these look like they might be resolved by using `text()` in place of `html()`, but I didn't actually review the code.
```
Linting common/static/common/js/discussion/views/discussion_topic_menu_view.js:
common/static/common/js/discussion/views/discussion_topic_menu_view.js: 39:25: javascript-jquery-html:                 this.$el.html(_.template($('#topic-template').html())(context));
common/static/common/js/discussion/views/discussion_topic_menu_view.js: 130:42: javascript-jquery-html:                 return this.selectedTopic.html(this.fitName(text));
common/static/common/js/discussion/views/discussion_topic_menu_view.js: 162:19: javascript-jquery-html:                 }).html(name).appendTo(document.body);
common/static/common/js/discussion/views/discussion_topic_menu_view.js: 162:30: javascript-jquery-insert-into-target:                 }).html(name).appendTo(document.body);
```